cider/exec/nextgen/jitlib/llvmjit/LLVMJITModule.cpp

```diff
@@ -91,4 +114,21 @@ void* LLVMJITModule::getFunctionPtrImpl(LLVMJITFunction& function) {
   }
   return nullptr;
 }
+
+void LLVMJITModule::addExternalModules(JITModule& external_module) {
+  external_module_ = &external_module;
+  copyExternalModule();
+}
+
+void LLVMJITModule::copyExternalModule() {
+  auto jit_external_module = reinterpret_cast<LLVMJITModule*>(external_module_);
+  module_ = llvm::CloneModule(
```

Comment:
will this method overwrite `module_`

Reply:
It will overwrite ```module_```, if we need to support muti-module, the function should be rewrite. But it maybe more complicated to maintain muli-module, and we may not have such demands in recent future?

Reply:
To my understanding, module is equal concepted to a query fragment. Then, multi-frag should be handled at higher level. Thoughts?

Reply:
Yes, one query fragment will only have one module.